<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeNullityWidget</class>
 <widget class="QWidget" name="ChangeNullityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QCheckBox" name="allAttributesCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add/remove filter value on all attributes of this table</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="allTablesCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add/remove filter value on all attributes of selected database</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>Affected Tables due to Inheritance</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="CustomSelector" name="filterCustomSelectorWidget" native="true"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Schema</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="schemaComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tableNameLabel">
       <property name="text">
        <string>Table Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tableComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="attributeNameLabel">
       <property name="text">
        <string>Attribute Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="attributeComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="singleValueLabel">
       <property name="text">
        <string>Value to alter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="singleValueComboBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="actionComboBox">
       <item>
        <property name="text">
         <string>Select operation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Add to Filter (Leave empty if filter is empty)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Add to Filter (Add value to empty filter)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Remove from Filter</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomSelector</class>
   <extends>QWidget</extends>
   <header>DsgTools.gui.CustomWidgets.SelectionWidgets.customSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
